cmd/tracee-ebpf/internal/flags/flags-capture.go

```diff
@@ -21,16 +21,17 @@ Possible options:
 [artifact:]mem                     capture memory regions that had write+execute (w+x) protection, and then changed to execute (x) only.
 [artifact:]net=interface           capture network traffic of the given interface. Only TCP/UDP protocols are currently supported.
 
-dir:/path/to/dir        path where tracee will save produced artifacts. the artifact will be saved into an 'out' subdirectory. (default: /tmp/tracee).
-profile                 creates a runtime profile of program executions and their metadata for forensics use.
-clear-dir               clear the captured artifacts output dir before starting (default: false).
+dir:/path/to/dir                    path where tracee will save produced artifacts. the artifact will be saved into an 'out' subdirectory. (default: /tmp/tracee).
+profile                             creates a runtime profile of program executions and their metadata for forensics use.
+clear-dir                           clear the captured artifacts output dir before starting (default: false).
+net:[per-container|per-process]     capture separate pcap file based on container/process context (default: none - saving one pcap for the entire host).
 
 Examples:
   --capture exec                                           | capture executed files into the default output directory
   --capture exec --capture dir:/my/dir --capture clear-dir | delete /my/dir/out and then capture executed files into it
   --capture write=/usr/bin/* --capture write=/etc/*        | capture files that were written into anywhere under /usr/bin/ or /etc/
   --capture profile                                        | capture executed files and create a runtime profile in the output directory
-  --capture net=eth0                                       | capture network traffic of eth0
+  --capture net=eth0 --capture net:per-container           | capture network traffic of eth0, and save pcap for each container
```

Comment:
Sorry for jumping in on this - IMO the arguments for net options are rather confusing. Both have 'net' prefix, but one time it's with '=' for the network, and one time with ':' for the pcap.
Wouldn't the argument for the pcap should be 'pcap'? like so:
--capture net=eth0 --capture **pcap**=[per-container|per-process]

Reply:
yes you're right its more accurate. i used `pcap:` instead of `pcap=` because this is more an option rather than a value (although i don't think it is consistent throughout the project)